src/gcp_scanner/crawl.py

```diff
@@ -961,12 +922,9 @@ def get_service_accounts(project_name: str,
     request = service.projects().serviceAccounts().list(name=name)
     while request is not None:
       response = request.execute()
-
-      for service_account in response.get("accounts", []):
-        service_accounts.append(
-            (service_account["email"], service_account["description"]
-             if "description" in service_account else ""))
-
+      service_accounts=[(service_account["email"],
```

Comment:
Whitespaces before and after `=`

Reply:
@mshudrak Hi, I have made the requested change. For '='  space is added before and after. But for extra space between list comprehension and 'request = ' line we kept it so that readability doesn't become an issue. Also for service_account list comprehension, it is split to multiple lines since lint check was causing the issue.